lib/rails_cloud_tasks/job.rb

```diff
@@ -60,17 +68,25 @@ def enqueue_task(payload, timestamp = nil)
           begin
             client.create_task(parent: queue_path, task: create_task(payload, timestamp))
           rescue Google::Cloud::FailedPreconditionError
-            client.create_queue(
-              parent: queue_path.split('/queues').first,
-              queue:  { name: queue_path }
-            )
+            client.create_queue(create_queue(queue_path))
 
             retry
           end
 
         response.name
       end
 
+      def create_queue(queue_path)
```

Comment:
You are not creating a queue here

```suggestion
      def build_queue(queue_path)
```